compiler/rustc_parse/src/parser/diagnostics.rs

```diff
@@ -71,8 +71,8 @@ impl Error {
                     "found a documentation comment that doesn't document anything",
                 );
                 err.help(
-                    "doc comments must come before what they document, maybe a comment was \
-                          intended with `//`?",
+                    "doc comments must come before what they document, if a comment was \
+                          intended use `//`",
```

Comment:
I personally think the new phrasing is slightly more terse (slightly shorter for an already long line of output) and removes the questioning tone of the help message. However, if the change does not add any new value, do let me know, then I can close this PR. Thanks.